yarn.lock

```diff
@@ -2139,10 +2139,10 @@
     "@elastic/transport" "^8.3.1"
     tslib "^2.4.0"
 
-"@elastic/elasticsearch@9.0.2":
-  version "9.0.2"
-  resolved "https://registry.yarnpkg.com/@elastic/elasticsearch/-/elasticsearch-9.0.2.tgz#66fb8465fdeabb0c1174d3c48c0a2d5ad5d177c4"
-  integrity sha512-uKA0PuPSND3OhHH9UFqnKZfxifAg/8mQW4VnrQ+sUtusTbPhGuErs5NeWCPyd/RLgruBWBmLSv1zzEv5GS+UnA==
+"@elastic/elasticsearch@9.0.3":
+  version "9.0.3"
+  resolved "https://registry.yarnpkg.com/@elastic/elasticsearch/-/elasticsearch-9.0.3.tgz#96302c6dfae27503d5271aba318883fdef4051bd"
+  integrity sha512-aagnssrVQi538wExO0Au169amtq68sXSwQMyzblQVAsqcmbqRTtzmGhKOjnDP0LK3ml0Mtje1uX+Vda7RhqDsA==
   dependencies:
     "@elastic/transport" "^9.0.1"
     apache-arrow "18.x - 19.x"
```

Comment:
Can we please bump this to 18 - 20? https://github.com/elastic/elasticsearch-js/blob/7be0f90526c3c4de51c7880bb8bed0edc3625889/package.json#L95

Won't affect Kibana but will help my type testing CI

Reply:
😮 AFAIK, it should list this from elasticsearch's dependencies. No idea why it kept 19.x 🫨 

cc @elastic/kibana-operations, do you have any idea?

Reply:
Nevermind, 9.03 is still using 19.x: https://github.com/elastic/elasticsearch-js/blob/9b595d5c0b5afcba8352c898d519db8444fd23df/package.json#L93-L97